wolfssl/ssl.h

```diff
@@ -1310,7 +1310,11 @@ WOLFSSL_API WOLFSSL_METHOD* wolfSSLv23_client_method(void);
 WOLFSSL_API WOLFSSL_METHOD* wolfSSLv2_client_method(void);
 WOLFSSL_API WOLFSSL_METHOD* wolfSSLv2_server_method(void);
 
+#if defined(WOLFSSL_QT)
```

Comment:
Switch on OPENSSL_VERSION_NUMBER instead of WOLFSSL_QT

Reply:
Guard by OPENSSL_VERSION_NUMBER instead of WOLFSSL_QT